feeds/wlan-ap/opensync/patches/11-protobuf-definitions

```diff
@@ -26,7 +26,7 @@ Index: opensync-2.0.5.0/interfaces/opensync_stats.proto
 +    CLIENT_DISCONNECT                       = 2;
 +    CLIENT_FAILURE                          = 3;
 +    CLIENT_FIRST_DATA                       = 4;
-+    CLIEND_ID                               = 5;
++    CLIENT_ID                               = 5;
```

Comment:
unrelated change

Reply:
This is a result of one of my forcefully pushed commits. This should be removed, I agree.